src/libraries/System.Net.Http/src/ILLink/ILLink.Suppressions.LibraryBuild.xml

```diff
@@ -141,6 +141,18 @@
       <property name="Target">M:System.Net.Http.HttpClientHandler.SetMaxAutomaticRedirections(System.Int32)</property>
       <property name="Justification">The Xamarin.iOS and Mono.Android libraries are not present when running the trimmer analysis during our build. A consuming application will get a warning if these libraries aren't present when trimming the full app.</property>
     </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
```

Comment:
@steveisok @mdh1418 - this is a pretty massive xml suppression file. It may make sense to make it `AnyMobile` specific and only use it when building for `AnyMobile` targets. That way we don't inadvertently suppress warnings when we shouldn't be.

Reply:
@mdh1418 Can you do a follow up PR w/ Eric's suggestion?